awesomeConfig.cmake

```diff
@@ -376,6 +376,12 @@ else()
                     ${BUILD_DIR}/${file}
                     COPYONLY)
     endforeach()
+
+    # There is no way to avoid that one
```

Comment:
What for / why?
Could not find "@@" in there.

And if required, this part could be moved into a separate file maybe?!